roomba_name: 1
num_s: 1.0
weight_s: 0.8
prob_s: 1.0
particle_num: 300
x_init: 7.5
y_init: 1.0
yaw_init: 1.57
bench: 1.0
fire: 1.0
big: 1.5
trash: 0.8
robot: 1.5
behind_roomba: 0.1
odom: 20
alpha_slow_th: 0.001
alpha_fast_th: 0.1
x_cov_init: 0.8
y_cov_init: 0.8
yaw_cov_init: 0.4
x_cov_1: 0.1
y_cov_1: 0.1
yaw_cov_1: 0.05
x_cov_2: 0.01
y_cov_2: 0.01
yaw_cov_2: 0.01
limit: 100
move_noise: 0.2
estimated_weight_th: 4.0
roomba_odom: roomba1_odom
is_only_odom: false
behind_th: 0.2
behind_roomba_th: 1.0
odom_topic_name: "/roomba1/roomba/odometry"
object_topic_name: "/roomba1/object_positions"
pose_topic_name: "/roomba1/db_pose"
poses_topic_name: "/roomba1/db_poses"
score_topic_name: "/roomba1/score"
path_topic_name: "/roomba1/path"
behind_topic_name: "/roomba2/fixed_score"
